bevy_nannou/src/main.rs

```diff
@@ -0,0 +1,82 @@
+use bevy::core_pipeline::clear_color::ClearColorConfig;
+use bevy::prelude::shape::Torus;
+use bevy::prelude::*;
+use bevy_nannou::NannouPlugin;
+use bevy_nannou_draw::color::{RED, SALMON, SEAGREEN, SEASHELL, SKYBLUE};
+
+pub fn main() {
+    App::new()
+        .add_plugins((DefaultPlugins, NannouPlugin))
+        .add_systems(Startup, startup)
+        .add_systems(Update, (update_draw, update_mesh))
+        .run();
+}
+
+#[derive(Resource, Deref, DerefMut)]
+struct MyTexture(Handle<Image>);
+
+fn startup(mut commands: Commands, assets: Res<AssetServer>, mut meshes: ResMut<Assets<Mesh>>) {
+    commands.insert_resource(AmbientLight {
+        color: Color::WHITE,
+        brightness: 1.0,
+    });
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Mesh::from(Torus {
+            radius: 100.0,
+            ..default()
+        })),
+        transform: Transform::from_xyz(-200.0, -200.0, 0.0),
+        ..default()
+    });
+
+    commands.spawn(
```

Comment:
we should probably spawn the camera for the user since we need to manage something like the orthographic settings and the clear color.